test/image_spec.rb

```diff
@@ -42,10 +39,14 @@
     it "is a HypriotOS" do
       expect(stdout).to contain('HypriotOS')
     end
+
+    it "is not dirty" do
+      expect(stdout).not_to contain('dirty')
```

Comment:
Seems that the root fs is "dirty", I have added a test and Travis build 92 https://travis-ci.org/hypriot/image-builder-rpi/builds/104843831 shows that there is a problem.

Refactored the other tests for better readability.